### PR TITLE
Update pggb to 0.5.3

### PR DESCRIPTION
**Pull request by BiocondaBot**
Update [`pggb`](https://bioconda.github.io/recipes/pggb/README.html): **0.5.2** &rarr; **0.5.3**

[install with bioconda](http://bioconda.github.io/recipes/pggb/README.html) [Conda](https://anaconda.org/bioconda/pggb/files)

Info | Link or Description
-----|--------------------
Recipe | [`recipes/pggb`](https://github.com//bioconda/bioconda-recipes/tree/bump/pggb/recipes/pggb) (click to view/edit other files)
Summary | This pangenome graph construction pipeline renders a collection of sequences into a pangenome graph (in the variation graph model).
Home | [https://github.com/pangenome/pggb](https://github.com/pangenome/pggb)
Releases |[https://github.com/pangenome/pggb/tags](https://github.com/pangenome/pggb/tags)
Recipe Maintainer(s) | @AndreaGuarracino
Author | `@pangenome`
***

This pull request was automatically generated (see [docs](https://bioconda.github.io/contributor/updating.html)).

### CODE REVIEW

**Review by AndreaGuarracino — APPROVED** [review]

**Comment by AndreaGuarracino** [comment]
@BiocondaBot please merge

File: recipes/pggb/build.sh [hunk 1]
```diff
@@ -7,3 +7,5 @@ sed -i ${n}'a\if [ $show_version == true ]; then echo "pggb '$PKG_VERSION'"; exi
 
 mkdir -p $PREFIX/bin
 mv pggb $PREFIX/bin
+mv partition-before-pggb $PREFIX/bin
+mv scripts/*.py $PREFIX/bin
```
Comment (subwaystation) [inline recipes/pggb/build.sh:11]: :fire:

Comment (subwaystation) [reply recipes/pggb/build.sh:11]: @AndreaGuarracino This should have brought the files in. I am confused.

Comment (AndreaGuarracino) [reply recipes/pggb/build.sh:11]: 9, it is moving the files, but `partition-before-pggb` wants the `scripts` folder.

**Pull request merged**